supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -82,6 +82,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedramp]]
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The _Federal Risk and Authorization Management Program (FedRAMP®)_ is managed by the FedRAMP Program Management Office. Add the registration symbol (®) on first use in a document. For more information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Comment:
How about:
```suggestion
*Description*: The _Federal Risk and Authorization Management Program (FedRAMP®)_ is managed by the FedRAMP Program Management Office. Write "FedRAMP®" with the registered trademark symbol on first use. For more information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Reply:
To me, the instruction 'Write "FedRAMP®" with the registered trademark symbol' is confusing. The quotes are not part of how FedRAMP® is used. Let's stick to language that is close to what the FedRAMP branding guidelines say, use the registration symbol with the first instance of the FedRAMP name.

Reply:
Hi @spank50, thank you for your feedback. I understand the confusion around the instructions for writing "FedRAMP®" with the registered trademark symbol. The intent, guided by [our contribution guidelines](https://github.com/redhat-documentation/supplementary-style-guide/blob/main/GUIDELINES.adoc), is to clarify the exact presentation of terms by using double quotation marks, as shown here:

```
Use double quotation marks ("") when emphasizing how to write a term. The following examples demonstrate the use of this guideline:
- Hyphenate "look-up" when using it as a modifier.
- Spell out "comma-separated values" on first use; use "CSV" thereafter.
```

Reply:
Feel free to adapt or ignore the suggestion as you see fit.